### DISCUSSION TITLE
Incorrect List of Contributors on Mobile App

### DISCUSSION THREAD

**Discussion by Faisal-Saleem** [community/community#142075] [Mobile]
One of my private repository is showing incorrect contributors. I have 2 external contributors and 1 pending invitation. on web its showing correct list of contributors but in mobile app it is showing 3 contributors but also incorrect. pending invitation is showing as contributor. 1 previous contributor who dont have access any more still showing as a contributor.

GitHub for iOS, Version 1.182.0 (144168013)
Device: iPhone 12 Pro Max (iOS 18.0.1)
</details>

**Comment by taki-on** [comment]
@Faisal-Saleem thank you for reporting the issue. We will file it as an issue and investigate. Thanks!

**Reply by Faisal-Saleem** [reply]
I think I made a mistake, On web I was looking into Collaborators but on Mobile app I was looking under Contributors. For private repo, web is not allowing me to see Contributors (Under insights) until I Upgrade to GitHub Pro or make this repository public to enable this feature. In mobile it is allowing me to see Contributors but no options to see collaborators.

**Comment by peachmontt** [comment]
It sounds like you're experiencing a bug with the GitHub iOS app regarding the display of contributors in your private repository. Here are some steps you can take to troubleshoot and potentially resolve this issue:

1. Refresh the App
Force Close and Reopen: Double-tap the home button (or swipe up from the bottom of the screen) and swipe up on the GitHub app to close it completely. Then reopen the app.
Pull to Refresh: On the repository screen, try pulling down to refresh the content.
2. Check Permissions
Make sure that the contributors in question still have access to the repository. If someone’s permissions have changed, the app may still cache their status incorrectly.
3. Check for App Updates
Ensure that you’re using the latest version of the GitHub app. Go to the App Store and check for any updates.
4. Clear Cache (if possible)
While the GitHub app doesn’t have a clear cache option, reinstalling the app can help. Before reinstalling, make sure to back up any data if necessary.
5. Reinvite Contributors
For the pending invitation, try revoking and reinviting the user to see if it updates correctly.
6. Contact GitHub Support
If the issue persists, it might be a bug in the app. You can report this issue directly to GitHub Support for a more detailed investigation. Provide them with:
The version of the app.
The device and iOS version.
A detailed description of the issue.
Screenshots, if possible.
7. Verify on Different Device
If you have access to another iOS device or an Android device, check if the same issue occurs there. This can help determine if the issue is device-specific or app-wide.
